central/reports/common/report_formatter.go

```diff
@@ -143,6 +152,7 @@ func Format(deployedImagesResults []DeployedImagesResult, watchedImagesResults [
 						v.FixedByVersion,
 						strings.ToTitle(stringutils.GetUpTo(v.Severity, "_")),
 						strconv.FormatFloat(v.Cvss, 'f', 2, 64),
+						strconv.FormatFloat(v.Nvdcvss, 'f', 2, 64),
```

Comment:
Don't we need to only add this when `includeNvd` is true?

More general question, could we just always include the NVD and if it is empty then we just don't have it?

Reply:
@dashrews78  this ads filed for select query but nvd cvss column is only added when flag is set to true 
https://github.com/stackrox/stackrox/pull/12870/files#diff-ac5eae65a138982ce2c2546e12b29d9f5d40e084f68acefb7d46dbd6419eb820R95

Reply:
That doesn't look like what the code is doing to me.  I'm not familiar with reports, but this looks like it is writing that field to the CSV for watched images.  Seems to me like you would need logic similar to what you have at line 124.

Reply:
Similar to how you did for deployed images, you might only want to add this if `includeNvd` is true

Reply:
+1